themes/Monokai Vibrant-color-theme.json

```diff
@@ -571,6 +571,7 @@
     "statusBar.debuggingBackground": "#21252B",
     "tab.activeBackground": "#2c313a",
     "tab.border": "#181A1F",
+		"tab.activeBorderTop": "#528BFF",
```

Comment:
I think `#007DD8` would be a better color choice here to match the existing selection highlight color. With that said, I do like this change.

Reply:
Agreed, good catch!